monitoring/kafka-connect/dashboard.py

```diff
@@ -0,0 +1,820 @@
+from grafanalib.core import (
+    TABLE_TARGET_FORMAT,
+    ConstantInput,
+    DataSourceInput,
+    Repeat,
+    RowPanel,
+    Table,
+    Template,
+    Templating,
+    Threshold,
+)
+
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import (
+    layout,
+    metrics,
+    Dashboard,
+    PieChart,
+    Stat,
+    Target,
+    TimeSeries,
+)
+
+
+class Metrics:
+    connector_status = metrics.Metric(
+        'kafka_connect_connector_status', 'status', 'instance', job='${job}'
```

Comment:
```suggestion
        'kafka_connect_connector_status', 'status', 'instance', job='${job}', namespace='${namespace}'
```

same for the other metrics